nms/app/views/network/__tests__/NetworkTest.tsx

```diff
@@ -270,32 +269,42 @@ describe('<NetworkDashboard />', () => {
   };
 
   beforeEach(() => {
-    axiosMock.post.mockImplementation(() =>
-      Promise.resolve({data: {success: true}}),
-    );
-    MagmaAPIBindings.putLteByNetworkId.mockImplementation(() =>
-      Promise.resolve({data: {success: true}}),
-    );
-    MagmaAPIBindings.putLteByNetworkIdCellularEpc.mockImplementation(() =>
-      Promise.resolve({data: {success: true}}),
-    );
-    MagmaAPIBindings.putLteByNetworkIdCellularRan.mockImplementation(() =>
+    (axiosMock as jest.Mocked<typeof axios>).post.mockImplementation(() =>
       Promise.resolve({data: {success: true}}),
     );
-    MagmaAPIBindings.putLteByNetworkIdDns.mockImplementation(() =>
-      Promise.resolve({data: {success: true}}),
-    );
-    MagmaAPIBindings.getNetworks.mockImplementation(() => Promise.resolve([]));
+    jest
+      .spyOn(MagmaAPI.lteNetworks, 'lteNetworkIdPut')
+      .mockImplementation(() =>
+        Promise.resolve({data: {success: true}} as AxiosResponse),
+      );
+    jest
+      .spyOn(MagmaAPI.lteNetworks, 'lteNetworkIdCellularEpcPut')
+      .mockImplementation(() =>
+        Promise.resolve({data: {success: true}} as AxiosResponse),
+      );
+    jest
+      .spyOn(MagmaAPI.lteNetworks, 'lteNetworkIdCellularRanPut')
+      .mockImplementation(() =>
+        Promise.resolve({data: {success: true}} as AxiosResponse),
+      );
+    jest
+      .spyOn(MagmaAPI.lteNetworks, 'lteNetworkIdDnsPut')
+      .mockImplementation(() =>
+        Promise.resolve({data: {success: true}} as AxiosResponse),
+      );
+    jest
+      .spyOn(MagmaAPI.networks, 'networksGet')
+      .mockImplementation(() => Promise.resolve({data: []} as AxiosResponse));
   });
 
   afterEach(() => {
-    axiosMock.get.mockClear();
-    MagmaAPIBindings.getLteByNetworkId.mockClear();
-    MagmaAPIBindings.getNetworksByNetworkId.mockClear();
-    MagmaAPIBindings.putLteByNetworkId.mockClear();
-    MagmaAPIBindings.putLteByNetworkIdCellularEpc.mockClear();
-    MagmaAPIBindings.putLteByNetworkIdCellularRan.mockClear();
-    MagmaAPIBindings.putLteByNetworkIdDns.mockClear();
+    (axiosMock as jest.Mocked<typeof axios>).get.mockClear();
+    (MagmaAPI.lteNetworks.lteNetworkIdGet as jest.Mock).mockClear();
+    (MagmaAPI.networks.networksNetworkIdGet as jest.Mock).mockClear();
+    (MagmaAPI.lteNetworks.lteNetworkIdPut as jest.Mock).mockClear();
+    (MagmaAPI.lteNetworks.lteNetworkIdCellularEpcPut as jest.Mock).mockClear();
+    (MagmaAPI.lteNetworks.lteNetworkIdCellularRanPut as jest.Mock).mockClear();
+    (MagmaAPI.lteNetworks.lteNetworkIdDnsPut as jest.Mock).mockClear();
```

Comment:
Will this be in conflict with https://github.com/magma/magma/pull/12718?

Reply:
Yes it is merged by now